optuna/pruners/threshold.py

```diff
@@ -0,0 +1,59 @@
+from optuna.pruners import BasePruner
+
```

Comment:
Just wondering but does the style checker require this blank line?

Reply:
Sorry, it is not needed. (removed)